addons/l10n_pe/models/res_city.py

```diff
@@ -0,0 +1,11 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# Copyright 2019 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
```

Comment:
remove all the vauxoo's copyright, this will be part of odoo's base code.

Reply:
@CarmenMiranda this can help you

```bash
grep -rl --exclude-dir=.git 'Copyright 2019 Vauxoo' |xargs sed -i 's/# Copyright 2019 Vauxoo.*/# Part of Odoo. See LICENSE file for full copyright and licensing details./'g

grep -rl --exclude-dir=.git 'License LGPL-3.0' | xargs sed -n -i '/License LGPL-3.0 or later.*/!p'

```

Reply:
@umiphos Thank you

Reply:
Done